package.json

```diff
@@ -234,7 +244,7 @@
 			],
 			"sourcery.editor": [
 				{
-					"command": "sourcery.rule.create",
+					"command": "sourcery.scan.open",
 					"title": "Create Sourcery rule with this pattern",
 					"when": "editorLangId==python && editorHasSelection"
```

Comment:
changed the right-click option when selecting code in the editor to open the scan and copy the code into the pattern, rather than creating a rule directly.